packages/db-react/src/use-db-network-live.tsx

```diff
@@ -1,7 +1,15 @@
 import { db } from '@workspace/db/db'
+import { dbNetworkFindMany } from '@workspace/db/db-network-find-many'
 import type { Network } from '@workspace/db/entity/network'
 import { useLiveQuery } from 'dexie-react-hooks'
+import { useRouteLoaderData } from 'react-router'
+import type { DbLoaderData } from './db-loader.tsx'
 
 export function useDbNetworkLive() {
-  return useLiveQuery<Network[], Network[]>(() => db.networks.orderBy('name').toArray(), [], [])
+  const data = useRouteLoaderData<DbLoaderData>('root')
```

Comment:
thought: we call this `useRouteLoaderData<DbLoaderData>('root')` 3 times.

How about we wrap it in a `useRootLoaderData()` hook?

Reply:
My preference would be we don't. I think the signature is already clear enough on what it does and I am not sure what we would gain from an abstraction here.